chainlink-testing-framework/setup-go/action.yml

```diff
@@ -28,7 +28,7 @@ runs:
   using: composite
   steps:
     - name: Setup Go
-      uses: actions/setup-go@v4
+    - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
```

Comment:
You have an extra dash here.

Reply:
Good catch - fixed in b816b7135fdc42b94bd00a0d552f65a00ffa91e7